### PR TITLE
Added performance data to plugin return

### PR DESCRIPTION
**Pull request by BarnesJeff**
Added performance data to plugin return. Display count for up, down+in and down+out osds

### CODE REVIEW

File: src/check_ceph_osd [hunk 1]
```diff
@@ -126,6 +126,7 @@ def main():
     print "Up OSDs: " + " ".join(up)
     print "Down+In OSDs: " + " ".join(down_in)
     print "Down+Out OSDs: " + " ".join(down_out)
+    print "| 'up'=%d 'down_in'=%d;;%d 'down_out'=%d;;%d" % (len(up), len(down_in), args.crit, len(down_out), args.crit)
```
Comment (valerytschopp) [inline src/check_ceph_osd]: Are you sure the single quotes around the perfdata names are required? (i.e. `'up'=...` vs `up=...`)

And the perfdata names could be prefixed with `osd_`. e.g. `osd_up`, `osd_down_in`, `osd_down_out`

File: src/check_ceph_osd [hunk 2]
```diff
@@ -136,6 +137,7 @@ def main():
...
```
Comment (valerytschopp) [inline src/check_ceph_osd]: Same comments

**Comment by BarnesJeff** [comment]
With regards to quotes around performance data labels:

According to nagios plugin guide https://nagios-plugins.org/doc/guidelines.html#AEN200, "the single quotes for the label are optional. Required if spaces are in the label"

Can remove quotes if required

**Comment by valerytschopp** [comment]
Hi Jeff,
Thanks for perfdata addition.

**Pull request merged**